cub/cub/device/dispatch/kernels/transform.cuh

```diff
@@ -439,17 +485,21 @@ _CCCL_DEVICE void transform_kernel_ublkcp(
       // TODO(ahendriksen): this could only have ptx::sem_relaxed, but this is not available yet
       ptx::mbarrier_arrive_expect_tx(ptx::sem_release, ptx::scope_cta, ptx::space_shared, &bar, total_copied);
     }
-
-    // all threads wait for bulk copy
-    __syncthreads();
-    while (!ptx::mbarrier_try_wait_parity(&bar, 0))
-      ;
   }
   else
   {
-    // use all threads to schedule an async_memcpy
-    int smem_offset = 0;
+    const bool elected = elect_one();
```

Comment:
If we elect a thread with id > BulkCopyAlignment, then there is no divergence later when we collectively copy the head and tail regions (or the small input early exit).